backend/src/providers/storage/s3.provider.ts

```
@@ -92,6 +100,30 @@ export class S3StorageProvider implements StorageProvider {
+    return this.parentAppKey ? `${this.parentAppKey}/${bucket}/${key}` : null;
+  }
+
+  /**
+   * Branch fallback wrapper for read paths: try the branch's S3 key first;
+   * on a "not found" miss (provider returned null), retry with the parent's
+   * S3 key when one is configured. Errors other than NotFound propagate.
+   */
+  private async withFallback<T>(
+    branchPath: string,
+    parentPath: string | null,
+    op: (s3Key: string) => Promise<T | null>,
```

ESLint

[error] Delete `,`

(prettier/prettier)